client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -261,6 +262,39 @@ const MobileNav = () => {
   );
 };
 
+const StuffMenu = () => {
+  const { isOpen, handlers } = useMenuProps('stuff');
+  const { newSketch } = useSketchActions();
+
+  const [createCollectionVisible, setCreateCollectionVisible] = useState(false);
+
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      <IconButton icon={AddIcon} {...handlers} />
+      <ul className={isOpen ? 'opened' : ''}>
+        <ParentMenuContext.Provider value="stuff">
+          <NavMenuItem onClick={() => newSketch()}>
+            {t('DashboardView.NewSketch')}
+          </NavMenuItem>
+          <NavMenuItem onClick={() => setCreateCollectionVisible(true)}>
+            {t('DashboardView.CreateCollection')}
+          </NavMenuItem>
+        </ParentMenuContext.Provider>
+      </ul>
+      {createCollectionVisible && (
+        <Overlay
+          title={t('DashboardView.CreateCollectionOverlay')}
+          closeOverlay={() => setCreateCollectionVisible(false)}
+        >
+          <CollectionCreate />
```

Comment:
Very nice re-use of the existing `<CollectionCreate />` component 👍 

I'm just thinking to myself here.  Somewhere down the line this could be refactored in such a way that the existing overlay handling in `DashboardView` applies to both mobile and desktop.  That's way in the future, a few steps past `// TODO: remove the switch and use a props like mobileTitle <Nav layout=“dashboard” mobileTitle={t(‘Login’)} />`.  Where we have multiple separate Nav components like `StuffNav`, `EditorNav`, etc. and can pass down an `onClickCollectionCreate` prop to `StuffNav` only.

Reply:
yeah that's a nice idea!